neqo-http3/src/recv_message.rs

```diff
@@ -29,7 +29,7 @@ const PSEUDO_HEADER_AUTHORITY: u8 = 0x8;
 const PSEUDO_HEADER_PATH: u8 = 0x10;
 const REGULAR_HEADER: u8 = 0x80;
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

I'm not sure about when we should use Eq, but it seems like this is a good case for that.  And this type really needs to implement Copy.